chapter2/tests/t2/chapter2-t2-gen-kustomization.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+#
+# Task 2 - Generate kustomization.yaml
+set -eo pipefail
+
+
+# Error handling function.
+err() {
+  echo "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: $*" >&2
+}
+
+
+# Checks (perms, args, dir-exist)
+if ! [[ -r $(pwd) ]]; then
+  err "Directory is not readable."
+  exit 255
+fi
+
+if [[ "$#" -lt 1 ]]; then
+  err "At least 1 or more args are expected"
+  exit 1
+fi
+
+if [[ ! -d "repos" ]]; then
+  mkdir repos
+fi
+
+
+# ssh-keys generation
+for arg in $@; do
```

Comment:
Let's add a better name instead of `arg`, i.e. `repo_name` and in below loop too, this will make your code easier to read and understand